### DISCUSSION TITLE
BC Law Society Member Card Credential

### DISCUSSION THREAD

**Discussion by cvarjao** [bcgov/bc-wallet-mobile#370] [Use Cases]
Issuer: Law Society of BC
- There can only be 1 active non-revoked credential at the time

Verifier: Court Services
- Proof Request Predicates
  - Must be Issued by Law Society of BC
  - Must not be revoked
- Connectionless proof-request

Verifier: BC Wallet Demo

### TEST
- ledger: [sovrin-staging](https://indyscan.io/txs/SOVRIN_STAGINGNET/domain?page=1&pageSize=50&filterTxNames=[%22NYM%22,%22SCHEMA%22,%22CLAIM_DEF%22]&sortFromRecent=true&search=%22AuJrigKQGRLJajKAebTgWu%22)
- [DID/NYM](https://indyscan.io/tx/SOVRIN_STAGINGNET/domain/106028) (`AuJrigKQGRLJajKAebTgWu`)
- [Schema ](https://indyscan.io/tx/SOVRIN_STAGINGNET/domain/209526) (`AuJrigKQGRLJajKAebTgWu:2:Member Card:1.5.1`)
- [Credential Definition](https://indyscan.io/tx/SOVRIN_STAGINGNET/domain/209527) (`AuJrigKQGRLJajKAebTgWu:3:CL:209526:default`)
### PROD
- ledger: [sovrin-main-net](https://indyscan.io/txs/SOVRIN_MAINNET/domain?page=1&pageSize=50&filterTxNames=[%22NYM%22,%22SCHEMA%22,%22CLAIM_DEF%22]&sortFromRecent=true&search=%224xE68b6S5VRFrKMMG1U95M%22)
- [DID/NYM](https://indyscan.io/tx/SOVRIN_MAINNET/domain/54627) (`4xE68b6S5VRFrKMMG1U95M`)
- [Schema ](https://indyscan.io/tx/SOVRIN_MAINNET/domain/59232) (`4xE68b6S5VRFrKMMG1U95M:2:Member Card:1.5.1`)
- [Credential Definition](https://indyscan.io/tx/SOVRIN_MAINNET/domain/59233) (`4xE68b6S5VRFrKMMG1U95M:3:CL:59232:default`)

**Comment by wadeking98** [comment]
Issuer: BC Wallet Demo

Features:

* Get a demo LSBC member card
* Use demo credential or you real LSBC credential in proof request
* Connectionless proof request

## Environments
#### DEV:
URL: https://bc-wallet-demo-dev.apps.silver.devops.gov.bc.ca/digital-trust/showcase/lawyer
Ledger: [CANdy-dev](https://candyscan.idlab.org/txs/CANDY_DEV/domain)
DID/NYM: `Trx3R1frdEzbn34Sp1jyX`
Schema: `Trx3R1frdEzbn34Sp1jyX:2:Member Card:1.5.1`
Credential Defenition: `Trx3R1frdEzbn34Sp1jyX:3:CL:26514:Member Card`

#### TEST:
URL: https://bc-wallet-demo-test.apps.silver.devops.gov.bc.ca/digital-trust/showcase/lawyer
Ledger: [CANdy-dev](https://candyscan.idlab.org/txs/CANDY_DEV/domain)
DID/NYM: `63ZiwyeZeazA6AhYRYm2zD`
Schema: `63ZiwyeZeazA6AhYRYm2zD:2:Member Card:1.5.1`
Credential Defenition: `63ZiwyeZeazA6AhYRYm2zD:3:CL:26676:Member Card`

#### PROD:
URL: https://digital.gov.bc.ca/digital-trust/showcase/lawyer
Ledger: [CANdy-dev](https://candyscan.idlab.org/txs/CANDY_DEV/domain)
DID/NYM: `XUxBrVSALWHLeycAUhrNr9`
Schema: `XUxBrVSALWHLeycAUhrNr9:2:Member Card:1.5.1`
Credential Defenition: `XUxBrVSALWHLeycAUhrNr9:3:CL:27369:Member Card`
